server/views/pages/bookAVisit/confirmation.njk

```diff
@@ -50,11 +50,12 @@
         {% endif %}
 
         {{ govukButton({
-            classes: "govuk-!-margin-top-3, govuk-!-margin-bottom-6",
+            classes: "govuk-!-margin-top-3, govuk-!-margin-bottom-4",
             text: "Go to manage prison visits",
             href: "/",
             preventDoubleClick: true
         }) }}
+        <br>
```

Comment:
Being a pain for you but `<br>` is for line breaks in text and not CSS layout, ideally want wrapping in a `<div>` or similar to stop them floating next to each other 🙂

Reply:
Fair point! Done :)